lambdas/section-manager-lambda/src/validators.ts

```diff
@@ -0,0 +1,11 @@
+import { assert } from 'typia';
+
+import { SqsSectionWithSectionItems } from './types';
+
+/**
+ * ensures data coming in via SQS satisfies the required structure and values
+ * @param sqsData
+ */
+export const validateSqsData = (sqsData: any): void => {
```

Comment:
I see that you took the `validateImageUrl` function and placed it in the `lambda-common` package,  nice! but I don't see the `SectionItem` `imageUrl` getting validated, is this happening in another PR?

Reply:
i _knew_ i was forgetting something in all this refactoring. will add this.

Reply:
oh wait - this has to happen down the road a bit, as the value can either come from SQS (ML) or the Parser (and i haven't gotten to implementing the graph calls yet).